src/lib.rs

```diff
@@ -48,3 +49,41 @@ pub use crate::ser::{
     pythonize, pythonize_custom, PythonizeDefault, PythonizeDictType, PythonizeListType,
     PythonizeTypes, Pythonizer,
 };
+pub use crate::visitor::PyObjectVisitor;
+
+#[cfg(feature = "serde_with")]
+/// This module provides a Serde `Serialize` and `Deserialize` implementation for `PyObject`.
+///
+/// ```rust
+/// use pyo3::PyObject;
+/// use serde::{Serialize, Deserialize};
+///
+/// #[derive(Serialize, Deserialize)]
+/// struct Foo {
+///   #[serde(with = "pythonize::pyobject")]
+///   #[serde(flatten)]
+///   inner: PyObject,
+/// }
+/// ```
+pub mod pyobject {
+    use pyo3::{PyObject, Python};
+    use serde::Serializer;
+
+    pub fn serialize<S>(obj: &PyObject, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        Python::with_gil(|py| {
+            let mut deserializer =
+                crate::Depythonizer::from_object_bound(obj.clone().into_bound(py));
+            serde_transcode::transcode(&mut deserializer, serializer)
+        })
+    }
+
+    pub fn deserialize<'de, D>(deserializer: D) -> Result<PyObject, D::Error>
+    where
+        D: serde::Deserializer<'de>,
+    {
+        Python::with_gil(|py| deserializer.deserialize_any(crate::PyObjectVisitor::new(py)))
```

Comment:
Should this also use `serde_transcode`? I think in that way the `PyObjectVisitor` might not be needed, and instead we use `Pythonizer` here in the same way we use `Depythonizer` above?